### ISSUE TITLE
json-diff cli never return exit code 1 in shell

### ISSUE THREAD

**Issue by neokree** [andreyvit/json-diff#88]
In my pipeline I was using json-diff to monitor different languages files. 
Now with the latest versions I always get success even if there are missing keys between files.

```
json-diff ....
 {
   container_form: {
+    required_fields: "The name of the {entity}, the warehouse and the unit are required fields"
   }
 }
echo $?
#=> 0
```

Is it possible to have the exit code not 0 when json-diff found some differences between files, like it was before?

**Comment by neokree** [comment]
I saw #51 and I think my script stopped working after using 0.6 

Maybe having an option for exit codes could solve the problem for everyone?
In the meantime I will revert back to 0.5.5 in my script

**Comment by ewoudenberg** [comment]
Thank you, Neokree, we'll look into it.

**Comment by ewoudenberg** [comment]
Resolved in 0.9.1
